lib/formatOrder.js

```diff
@@ -0,0 +1,16 @@
+var sorting = require('postcss-sorting');
+
+function formatOrder(root, params) {
+  var sortOrder = params.stylelint['declaration-block-properties-order'];
+  if (!Array.isArray(sortOrder)) {
```

Comment:
postcss-sorting also supports [grouping](https://github.com/hudochenkov/postcss-sorting#grouping), but this feature not supported by stylelint 😞